docs/misc/commands.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar-position: 7
+---
+# Commands
+Commands are very useful in vanilla Minecraft and even more so with mods. They are the quickest way to access and modify data.
+Commands allow to:
+- Use permission levels.
+- Access to anything in your mod.
+- Allow use anywhere on the map, generally without specific items or other physical objects.
+## Creating a command
+To create a command, you need to create a new class and put this code:
+```java
+public static void register(CommandDispatcher<CommandSourceStack> dispatcher) {
+    dispatcher.register (
+        Commands.literal("commandname") // The name of your command
+        .execute(context -> { // Here is what the command do, where context is a CommandContext<CommandSourceStack>
+            // Put action here
+            return Command.SINGLE_SUCCES; // send succes to server
+        })
```

Comment:
Ok, so the command does something, but what am I taking in, and what am I returning?

Reply:
You still haven't really explained what a 'success' is. Why is success 1? Can it be larger numbers? Smaller numbers? You need to explain.